### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by Jean-Beru**
| Q             | A
| ------------- | ---
| Branch?       | main
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | 
| License       | MIT
| Doc PR        | 

This MR makes CI green again. But, to make it possible, note that:
* `@statusCode` tag is skipped on Selenium tests since the driver doesn't support `getStatusCode` method
* `@rest` tag is skipped on Selenium tests since the driver doesn't support `getResponseHeaders` method
* `@json` and `@xml` tags are skipped on Selenium tests since the response is wrapped inside HTML tags

The two last points may not be a problem since this kind of tests doesn't need a Chrome nor Firefox to run since no javascript have to be executed.

### CODE REVIEW

**Comment by soyuka** [comment]
OMG huge thanks!!!

**Comment by ottaviano** [comment]
yeah, thanks @Jean-Beru for this work! 
@soyuka, would it be possible to create a new version with these patches?

**Comment by soyuka** [comment]
sure

**Comment by soyuka** [comment]
done

File: src/HttpCall/HttpCallListener.php [hunk 1]
```diff
@@ -51,9 +51,11 @@ public function afterStep(AfterStepTested $event)
         // For now to avoid modification on MinkContext
         // We add fallback on Mink
         try {
-            $this->httpCallResultPool->store(
-                new HttpCallResult($this->mink->getSession()->getPage()->getContent())
-            );
+            if ($this->mink->getSession()->isStarted()) {
```
Comment (soyuka) [inline src/HttpCall/HttpCallListener.php:54]: Note that this broke our test suite

**Comment by soyuka** [comment]
it looks like behat is now taking ~3hours and is broken lol

**Comment by Jean-Beru** [comment]
@soyuka Fixed in https://github.com/soyuka/contexts/pull/24

File: src/HttpCall/HttpCallListener.php (hunk 1 above)
Comment (walva) [reply src/HttpCall/HttpCallListener.php:54]: Same, and 2 other people experienced this issue too here : https://github.com/soyuka/contexts/commit/8215019f4f75d3e184f70b075e5a352b6551f0bc#r109969509

**Pull request merged**